src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -541,8 +542,8 @@ instance MetaOrObject level => PrettyPrint (CommonPurePattern level) where
     prettyPrint flags purePattern =
         writeOneFieldStruct flags "Fix" (unFix purePattern)
 
-instance PrettyPrint (Fix (pat variable))
-    => PrettyPrint (Attributes pat variable)
+instance PrettyPrint (CommonKorePattern)
```

Comment:
Same here;  it seems you shouldn't need an instance for `PrettyPrint CommonKorePattern`, as it should already be pretty-printable.

Reply:
Doesn't compile if I comment this out. Probably because Attributes is a newtype.